README.md

```diff
@@ -78,12 +76,13 @@ main
 
 All Kubernetes-CSI repos are expected to switch to Prow. For details
 on what is enabled in Prow, see
-https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes-csi
+<https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes-csi>
```

Comment:
Here the linter seems a bit too strict, given that GitHub (our main target) can handle plain embedded URLs just fine.

Reply:
True. but the extra brackets are only added in the README of this repo. so it should not cause extra churn in important repos unless we enable mdlinter. Even then we can define a skip rule.